ungrib/src/rrpr.F

```diff
@@ -1399,4 +1412,43 @@ subroutine fix_ruc_soilm (ix, jx)
 
 end subroutine fix_ruc_soilm
  
-
+subroutine fix_gefs_landmask (ix, jx, plvl)
+! Set LANDSEA to values based on soil temp. Assumes fix_gfs_miss has already been called.
+! Needed for September 23, 2020 changes to GEFS. 
+! Earlier GEFS files are unmodified. Must be called after fix_gfs_miss
+! plvl must always be 200100.
+!
+  use storage_module
+  use module_debug
+  implicit none
+  integer :: ix, jx, i, j, k
+  real :: plvl
+  real, allocatable, dimension(:,:) :: f, sea
+!
+! If LANDN is present (not currently the case) then exit
+  if ( is_there(200100, 'LANDN') ) then
+    return
+  else
+    allocate(sea(ix,jx))
+    allocate(f(ix,jx))
+    if (is_there(nint(plvl), 'ST000010' )) then
+      call get_storage(nint(plvl), 'ST000010', f, ix, jx)
+      call get_storage(nint(plvl), 'LANDSEA', sea, ix, jx)
+      do j = 1, jx
+      do i = 1, ix
+	if (f(i,j) .le. 0.) then
+	  sea(i,j) = 0.
+	else
+	  sea(i,j) = 1.
+	endif
```

Comment:
I think it would be good to use spaces rather than tabs to indent this if/else block.

Reply:
I will fix this too.